src/ui-test/tests/jbang_user_preference.test.ts

```diff
@@ -0,0 +1,169 @@
+            await input.confirm();
+
+            await driver.wait(async function () {
+                return (await new EditorView().getOpenEditorTitles()).find(t => t === FILENAME);
+            }, 220000);
+
+            await waitUntilTerminalHasText(driver, [`-Dcamel.jbang.version=${OLDER_JBANG_VERSION}`]);
+            expect(await (await activateTerminalView()).getText()).to.contain(`-Dcamel.jbang.version=${OLDER_JBANG_VERSION}`);
+
+            terminalView = await new BottomBarPanel().openTerminalView();
+            await terminalView.selectChannel('Init Camel Route file with JBang');
+            await terminalView.killTerminal();
+        });
+    });
+
+    async function extensionIsActivated(marketplace: Marketplace): Promise<boolean> {
+        try {
+            const item = await marketplace.findExtension(`@installed ${pjson.displayName}`);
+            const activationTime = await item.findElement(By.className('activationTime'));
+            if (activationTime !== undefined) {
+                return true;
+            } else {
+                return false;
+            }
+        } catch (err) {
+            return false;
+        }
+    }
+
+    async function getJBangVersion(): Promise<string> {
+        const textField = await (await new Workbench().openSettings()).findSetting('JBang Version', 'Camel', 'Language Support');
+        const value = await textField.getValue() as string;
+        await new EditorView().closeEditor('Settings');
+        return value;
+    }
+
+    async function setJBangVersion(version: string): Promise<void> {
+        const textField = await (await new Workbench().openSettings()).findSetting('JBang Version', 'Camel', 'Language Support');
+        await textField.setValue(version);
+        await driver.sleep(500);
+        await new EditorView().closeEditor('Settings');
+    }
+
+    async function waitUntilTerminalHasText(driver: WebDriver, textArray: string[], interval = 500): Promise<void> {
+        await driver.wait(async function () {
+            try {
+                const terminal = await activateTerminalView();
+                const terminalText = await terminal.getText();
+                for (const text of textArray) {
+                    if (!(terminalText.includes(text))) {
+                        return false;
+                    }
+                }
+                return true;
+            } catch (err) {
+                return false;
+            }
+        }, 220000, undefined, interval);
+    }
+
+    async function activateTerminalView(): Promise<TerminalView> {
+        // workaround ExTester issue - https://github.com/redhat-developer/vscode-extension-tester/issues/785
+        await new Workbench().executeCommand('Terminal: Focus on Terminal View');
+        return await new BottomBarPanel().openTerminalView();
+    }
+
+    function deleteFile(filename: string): void {
+        fs.remove(path.resolve(RESOURCES, filename), (err: any) => {
```

Comment:
`remove` is async.. you need to modify method to return Promise and use await or use `fs.removeSync` instead